### PR TITLE
Multus for vsphere

### PR DESCRIPTION
**Pull request by shyRozen**
Adding support for Multus in Vsphere.
Most of the changes is adding interface to each compute node and defining the network policy for vsphere.

### CODE REVIEW

File: ocs_ci/templates/ocs-deployment/node_network_configuration_policy.yaml [hunk 1]
```diff
@@ -1,29 +1,31 @@
 apiVersion: nmstate.io/v1
 kind: NodeNetworkConfigurationPolicy
 metadata:
- name: ceph-public-net-shim-worker-node
- namespace: openshift-storage
+  name: ceph-public-net-shim-worker-node
+  namespace: openshift-storage
```
Comment (OdedViner) [inline ocs_ci/templates/ocs-deployment/node_network_configuration_policy.yaml:5]: please delete the space.

Comment (shyRozen) [reply ocs_ci/templates/ocs-deployment/node_network_configuration_policy.yaml:5]: I think it somehow messed here in how github present it. This is how it looks in the editor:
image

**Review by OdedViner — APPROVED** [review]

File: ocs_ci/templates/ocs-deployment/node_network_configuration_policy.yaml (hunk 1 above)
Comment (shyRozen) [reply ocs_ci/templates/ocs-deployment/node_network_configuration_policy.yaml:5]: The indentation of the file was just one space and the editor fixed it to have 2 spaces which is our regular formatting of yaml

**Review by ebenahar — APPROVED** [review]

**Pull request merged**